lib/src/_query.dart

```diff
@@ -16,7 +16,7 @@ abstract class Query {
         if (must != null) 'must': must,
         if (filter != null) 'filter': filter,
         if (should != null) 'should': should,
-        if (mustNot != null) 'mustNot': mustNot,
+        if (mustNot != null) 'must_not': mustNot,
```

Comment:
Do you happen to know which version had this change? It would be nice if we could put that information in the changelog.

Reply:
I think that at least 0.90, we can use "must_not" as follows.

https://www.elastic.co/guide/en/elasticsearch/reference/0.90/query-dsl-bool-query.html

I could not find the specific version because it's too old.

Reply:
Oh wow, it is not really news-worthy then. Thanks for digging it up.